contrib/openvpn/files/update-resolv-conf

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
```

Comment:
this is bash but you didn't add depends=bash

you could port the script too

Reply:
that script could be reasonably easily rewritten without using bash and probably shorter and cleaner (e.g. it doesn't need to check where resolvconf is etc.)

Reply:
> When you say port it you mean to sh?

yeah, and you have to actually make some changes for that since it's not valid sh as-is